frontend/src/components/Router.tsx

```diff
@@ -31,6 +31,7 @@ import AllExperimentsAndArchive, {
   AllExperimentsAndArchiveTab,
 } from '../pages/AllExperimentsAndArchive';
 import AllRunsAndArchive, { AllRunsAndArchiveTab } from '../pages/AllRunsAndArchive';
+import AllJobsList from '../pages/AllRecurringRunsList';
```

Comment:
Should it be `AllRecurringRunsList` instead?

Reply:
@zijianjoy Good catch! Fixed it